crates/samples/kernel_event/src/main.rs

```diff
@@ -1,11 +1,12 @@
 use windows::{
+    core::PCWSTR,
     Win32::Foundation::CloseHandle,
     Win32::System::Threading::{CreateEventW, SetEvent, WaitForSingleObject, WAIT_OBJECT_0},
 };
 
 fn main() -> windows::core::Result<()> {
     unsafe {
-        let event = CreateEventW(std::ptr::null(), true, false, None)?;
+        let event = CreateEventW(std::ptr::null(), true, false, PCWSTR::default())?;
```

Comment:
Ideally, we'd have more support for `Option` rather than less. https://github.com/microsoft/windows-rs/issues/816